photon-core/src/main/java/org/photonvision/vision/processes/VisionRunner.java

```diff
@@ -53,15 +54,18 @@ public VisionRunner(
             FrameProvider frameSupplier,
             Supplier<CVPipeline> pipelineSupplier,
             Consumer<CVPipelineResult> pipelineResultConsumer,
-            QuirkyCamera cameraQuirks) {
+            QuirkyCamera cameraQuirks,
+            Supplier<VisionModuleChangeSubscriber> changeSubSupplier) {
         this.frameSupplier = frameSupplier;
         this.pipelineSupplier = pipelineSupplier;
         this.pipelineResultConsumer = pipelineResultConsumer;
         this.cameraQuirks = cameraQuirks;
+        this.changeSubSupplier = changeSubSupplier;
 
         visionProcessThread = new Thread(this::update);
         visionProcessThread.setName("VisionRunner - " + frameSupplier.getName());
         logger = new Logger(VisionRunner.class, frameSupplier.getName(), LogGroup.VisionModule);
+        changeSubSupplier.get().processSettingChanges();
```

Comment:
Any particular reason you don't have VisionRunner own the VisionModuleChangeSubscriber if it needs to know that it exists anyways?